p2p/backend-test-suite/src/peer_manager_peerdb.rs

```diff
@@ -20,124 +20,42 @@ use std::{sync::Arc, time::Duration};
 
 use p2p::{
     config::P2pConfig,
-    net::{
-        default_backend::types::PeerId,
-        types::{PeerInfo, PubSubTopic, Role},
-        AsBannableAddress, NetworkingService,
-    },
+    net::{default_backend::types::PeerId, AsBannableAddress, NetworkingService},
     peer_manager::peerdb::{
-        storage::{PeerDbStorage, PeerDbStorageRead, PeerDbTransactional},
+        storage::{PeerDbStorageRead, PeerDbTransactional},
         PeerDb,
     },
-    testing_utils::{peerdb_inmemory_store, RandomAddressMaker},
+    testing_utils::{peerdb_inmemory_store, P2pTestTimeGetter, RandomAddressMaker},
 };
 
-tests![
```

Comment:
Where did these tests go?

Reply:
These two specific tests have been removed. Not much is done there, and `adjust_peer_score` calls (below and above threshold) are tested in `p2p/src/peer_manager/tests/ban.rs`.